presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -1210,6 +1211,15 @@ public Void visitLateralJoin(LateralJoinNode node, Void context)
             return processChildren(node, context);
         }
 
+        @Override
+        public Void visitNativeExecution(NativeExecutionNode node, Void context)
+        {
+            addNode(node, "NativeExecution", "", ImmutableList.of(node.getSubPlan()));
```

Comment:
nit: Can we add an additional overloaded addNode() method that takes all above parameters except for identifier. Refer to addNode(PlanNode node, String name). Thought it will be cleaner but no strong opinon